### ISSUE TITLE
Alterations in presentation of legacy data - medium priority

### ISSUE THREAD

**Issue by alieden** [Clinical-Genomics/statina#189]
The different groups of previously analyzed samples should not overlap so that data is missing. For example, it's not possible to see the verified T13 sample in the Zscore 13 graph.  

We would also suggest that the number in each group is put within brackets to improve the clarity i.e. "Verified T13 (#1)" for example? The same goes for all graphs with legacy data.

The number for the negative samples (legacy data) in the graphs is incorrect at the moment (says 46). In the Zscore 13 graph this number should be all included samples that are negative for T13 i.e. that are not manually classified into a T13 group such as suspected/probable/verified and so on.

**Comment by alieden** [comment]
Better if the number in each group is presented in the same way as in the previous version of statina, i.e. "Verified T13 (n=1)"

**Comment by mayabrandi** [comment]
I dont understad what you mean. It is possible to  verified T13 sample in the Zscore 13 graph. And there is no overlap. What data are you missing. can you give an example.




![Skärmavbild 2022-01-28 kl  15 40 44](https://user-images.githubusercontent.com/1306333/151566703-1ab4640e-f7b6-42b7-b892-c67f5444d445.png)

**Comment by alieden** [comment]
The presentation has now been changed. The verified case was behind the suspected cases. Anyway, I think we should have similar presentation as in the old viewer?
![NCV13](https://user-images.githubusercontent.com/22740092/151751877-fe9b0d84-764d-415b-ae5b-6cff476bcc3f.png)

**Comment by mayabrandi** [comment]
Aha. Now i understand what you mean. Nothing has changed with the control samples in the plots since you wrote the issue. You know you can hide samples by clicking at them in the legend!?

**Comment by alieden** [comment]
That's right, the verified sample was probably hidden by a click (although I don't recall that). I would anyway like to have the groups (verified, false positive etc) separated since this is clearer whent you show the data in a figure for a presentation etc.
